src/lib/dialog/dialog.ts

```diff
@@ -65,6 +67,10 @@ export class MdDialog {
     let dialogRef =
         this._attachDialogContent(componentOrTemplateRef, dialogContainer, overlayRef, config);
 
+    if (!this._openDialogs.length && !this._parentDialog) {
+      document.addEventListener('keydown', this._boundKeydown);
```

Comment:
I guess we don't have access to a Renerer we can use to do this at this point in the code?